assets/xml/overlays/ovl_En_Holl.xml

```diff
@@ -0,0 +1,9 @@
+<Root>
+    <File Name="ovl_En_Holl" BaseAddress="0x80899960" RangeStart="0x0BF0" RangeEnd="0x0C58" Segment="128">
+        <Array Name="gEnHollUncoloredRectangleVertices" Count="4" Offset="0x0BF0">
```

Comment:
Extracted Vtx must be suffixed by `Vtx`
```suggestion
        <Array Name="gEnHollUncoloredRectangleVtx" Count="4" Offset="0x0BF0">
```

Reply:
Oh okay. I was just thinking Vertices was more grammatically correct since it was an array. I'll change it. I need to change it in the code too, so I'll just do a push to avoid 🚫 🆗

Reply:
I think it would be clearer to just name these based on how they're used rather than what it looks like. So @AngheloAlf 's previous suggestion using `Plane` seems better IMO. Also, `Vtx` is the standard suffix we use for vertices, like `DL` for DisplayList. There's a list in the ZAPD docs or the object decomp tutorial.

Reply:
Great.
If you are curious we also have some small docs for this kind of stuff [here](https://github.com/zeldaret/mm/blob/master/tools/ZAPD/docs/zapd_extraction_xml_reference.md).

Reply:
Fixed in https://github.com/zeldaret/mm/pull/270/commits/4f881b73f87a6119a54989296129baeb3e88e405

Reply:
Just making sure, but did you saw @EllipticEllipsis comment?

Reply:
Oh i did not. I hadn't refreshed my browser.

Reply:
I thought I was naming according to how it's used. The DL uses the uncolored rectangle from the Vtx to convert into the central rectangle of the Holl.

Reply:
Yes, that's the point, if you look at what it ends up doing, it's the central plane. So I suggest naming it as such. `UncoloredRectangle` doesn't tell you what it does, whereas `CentralPlane` would.

Reply:
Is CentralRectangle fine? That's what I named the DL. It just slightly irks the math major in me to call this a plane when it's non-infinite. I can call the Vtx that too.

Reply:
I agree that it's not the correct definition of plane, but it wouldn't be the first or last time we went with incorrect mathematical terminology. You also *can't* have an actual displayed plane in graphics since computers can't do infinite stuff. I'd be okay with `Rectangle`, though.

Reply:
The planes in this actor are actually infinite though. They're just a single z coordinate, which describes a plane in X and Y. It's just the drawing itself that's non-infinite (for obvious reasons).

Reply:
In that case, `Plane` would be correct, right?

Reply:
I suppose if you think about it as the (inherently finite) graphical representation of the (inherently infinite) plane, then `Plane` would actually be an adequate name for it. It is honestly easier to understand that way too, so I changed it in https://github.com/zeldaret/mm/pull/270/commits/57528dd548a213cac8f416f00e2753e82ddb97d3. marking as resolved.